### PR TITLE
Move ResearchUI framework to Research project

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
This moves the ResearchUI framework as a framework built as a part of the Research project. This is to allow both of these frameworks to be version supported by Carthage, which is not set up to allow resolving dependencies for more than one framework within the same workspace but *not* the same project. It makes for a less ideal directory structure, but I think that with some additional documentation, it's not too hard to understand.

Next steps will be to set up BridgeApp to use Carthage for dependency management to unblock our partners that use Bridge.

Note: as a part of this build, to get Carthage working, I removed CRF and MotorControl from the schemas but did not remove them (yet) from the repo. That said, until I can finish setting those projects up to point to this project with the new structure, they shouldn't be modified here.

I also removed RSDTest b/c it seemed to be confusing people and its inclusion isn't needed with iOS 11.

### CODE REVIEW

File: Documentation/index.html [hunk 1]
```diff
@@ -44,15 +44,63 @@
...
-<h3 id='authors' class='heading'>Authors</h3>
-
-<p>Sage Bionetworks</p>
-
+            <h1 id='sageresearch' class='heading'>SageResearch</h1>
+            
+            <p><a href="http://sagebionetworks.org">Sage Bionetworks</a> has spent four years developing mobile health study apps and played a role in supporting over 20 projects, using Research Kit as the basis for all of our iPhone development. Based on that experience, we have begun to generate a next-generation framework and are looking for the involvement of the broader Research Kit community to provide input on design goals and approaches, and contribute to development. This framework:</p>
+            
+            <ul>
+                <li>Keeps what is good and useful about ResearchKit, provides for a smooth transition from it, and allows ResearchKit 1.x components to be used as modules in Research apps;</li>
```
Comment (Erin-Mounts) [inline Documentation/index.html:52]:
```suggestion
                <li>Keeps what is good and useful about ResearchKit, provides for a smooth transition from it, and allows ResearchKit 2.x components to be used as modules in Research apps;</li>
```

File: Documentation/index.html [hunk 2]
```diff
@@ -44,15 +44,63 @@
...
+                    documentation and/or other materials provided with the distribution.</li>
+                <li>Neither the name of Sage Bionetworks nor the names of BridgeSDk&rsquo;s
+                    contributors may be used to endorse or promote products derived from
+                    this software without specific prior written permission.</li>
+            </ul>
+            
+            <p>THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS <q>AS IS</q> AND
+            ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+            WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+            DISCLAIMED. IN NO EVENT SHALL <COPYRIGHT HOLDER> BE LIABLE FOR ANY
```
Comment (Erin-Mounts) [inline Documentation/index.html]:
```suggestion
            DISCLAIMED. IN NO EVENT SHALL SAGE BIONETWORKS BE LIABLE FOR ANY
```
Also the same edit should be made in README.md

File: README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 [Sage Bionetworks](http://sagebionetworks.org) has spent four years developing mobile health study apps and played a role in supporting over 20 projects, using Research Kit as the basis for all of our iPhone development. Based on that experience, we have begun to generate a next-generation framework and are looking for the involvement of the broader Research Kit community to provide input on design goals and approaches, and contribute to development. This framework:
 
 - Keeps what is good and useful about ResearchKit, provides for a smooth transition from it, and allows ResearchKit 1.x components to be used as modules in Research apps;
```
Comment (Erin-Mounts) [inline README.md:5]:
```suggestion
- Keeps what is good and useful about ResearchKit, provides for a smooth transition from it, and allows ResearchKit 2.x components to be used as modules in Research apps;
```

File: Research/Research-UnitTest/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+notice, this list of conditions and the following disclaimer in the
+documentation and/or other materials provided with the distribution.
+* Neither the name of Sage Bionetworks nor the names of BridgeSDk's
+contributors may be used to endorse or promote products derived from
+this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL <COPYRIGHT HOLDER> BE LIABLE FOR ANY
```
Comment (Erin-Mounts) [inline Research/Research-UnitTest/README.md]:
```suggestion
DISCLAIMED. IN NO EVENT SHALL SAGE BIONETWORKS BE LIABLE FOR ANY
```

**Review by Erin-Mounts — APPROVED** [review]
A few nits (that have been there all along I think), otherwise looks good

File: Documentation/index.html (hunk 1 above)
Comment (syoung-smallwisdom) [reply Documentation/index.html:52]: Yeah, I thought about that but I haven't really tested it against 2.x so ?

**Pull request merged**